src/acceptance/assets/app/go_app/internal/app/app.go

```diff
@@ -1,36 +1,49 @@
 package app
 
 import (
-	"log"
+	"context"
+	"github.com/cloudfoundry-community/go-cfenv"
 	"net/http"
 	"time"
 
+	ginzap "github.com/gin-contrib/zap"
 	"github.com/gin-gonic/gin"
-	"github.com/sirupsen/logrus"
+	"github.com/go-logr/zapr"
+	"go.opentelemetry.io/contrib/instrumentation/github.com/gin-gonic/gin/otelgin"
+	"go.opentelemetry.io/contrib/propagators/b3"
+	"go.opentelemetry.io/otel"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
 )
 
-type logrusErrorWriter struct{ *logrus.Logger }
+func Router(logger *zap.Logger, sleep func(duration time.Duration), useMem func(useMb uint64), useCPU func(utilization uint64, duration time.Duration), postCustomMetrics func(ctx context.Context, appConfig *cfenv.App, metricsValue float64, metricName string, useMtls bool) error) *gin.Engine {
```

Comment:
What about a line-break here as it already contains more than the maximum length of a twitter-tweet (i.e. close to 300 symbols)?

Reply:
Replaced hand-written mocking with fakes generated by counterfeiter